website/snippets/_v2-sl-prerequisites.md

```diff
@@ -0,0 +1,55 @@
+
+<VersionBlock firstVersion="1.6">
+
+- Have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America
+- Have both your production and development environments running dbt version 1.6 or higher 
+- Use Snowflake, BigQuery, Databricks, and Redshift data platform
+- A successful run in the environment where your Semantic Layer is configured
+  * Note &mdash; Deployment environment is currently supported (_development experience coming soon_)
+- Install the [MetricFlow CLI](/docs/build/metricflow-cli). After installing the package, make sure you run at least one model.
+- Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions 
+  * Developer accounts will be able to query manually using the [MetricFlow CLI](/docs/build/metricflow-cli) and SQL. To dynamically query metrics using external tools, you'll need to have a dbt Cloud Team or Enterprise account with access to the Semantic Layer API.<br />
+- Have an understanding of [MetricFlow's](/docs/build/about-metricflow) key concepts, which powers the revamped dbt Semantic Layer.
+
+
+</VersionBlock>
+
+
+<VersionBlock lastVersion="1.5">
```

Comment:
I don't see any difference between version 1.5 and 1.3 (unless I'm missing something?) Maybe we can combine them to make the code easier to read? To combine, you could change `<VersionBlock firstVersion="1.3" lastVersion="1.3" >` to `<VersionBlock firstVersion="1.3" lastVersion="1.5" >`